src/coreclr/System.Private.CoreLib/src/System/Runtime/CompilerServices/CastHelpers.cs

```diff
@@ -600,7 +600,7 @@ internal static void Unbox_Nullable(ref byte destPtr, MethodTable* typeMT, objec
                 }
                 else
                 {
-                    Unsafe.As<byte, bool>(ref destPtr) = true;
+                    Unsafe.WriteUnaligned(ref destPtr, true);
```

Comment:
Bools are one byte long so this is not needed; accessing them is always aligned.

Reply:
Here and below, can you replace these byte <-> bool with ternary ops and run MihuBot?